src/PubSubClient.h

```diff
@@ -121,6 +123,7 @@ class PubSubClient {
    PubSubClient& setCallback(MQTT_CALLBACK_SIGNATURE);
    PubSubClient& setClient(Client& client);
    PubSubClient& setStream(Stream& stream);
+   PubSubClient& setPacketBuffer(uint8_t * pBuf, uint16_t length = _MQTT_MAX_PACKET_SIZE);
```

Comment:
Having a default value here makes it easy for people to forget passing the length. Think it should be dropped, even if it means a bit of code duplication in the default constructors.